src/lib/ngsi/ContextAttributeVector.cpp

```diff
@@ -507,15 +339,15 @@ void ContextAttributeVector::fill(ContextAttributeVector* cavP, bool useDefaultT
 *
 * lookup -
 */
-ContextAttribute* ContextAttributeVector::lookup(const std::string& attributeName) const
+int ContextAttributeVector::lookup(const std::string& attributeName) const
```

Comment:
A better name of this method would be 'index' or 'getIndex'.
'lookup' indicates to actually return what was looked up ...

Reply:
Good point. Chaning name.

Reply:
Fixed in 295296b